GeraldWalravens.txt

```diff
@@ -0,0 +1 @@
+Hello World, this is geraldwal
```

Comment:
Please remove this file :) And keep only the `names/...` one

Reply:
Should be done. Thanks for your review and feedback!